src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    text-transform: uppercase;
+    font-size: 12px;
+  }
+
+  &__list :first-child {
+    margin: 0;
+  }
+  
+  &__item {
+    display: block;
+    margin-left: 20px;
+  }
+  
+  &__link {
+    display: block;
+    height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. Use `padding: 23px 0;` instead 
```suggestion
```